install_config/topics/ocp_osp_admin_tasks.adoc

```diff
@@ -1,7 +1,7 @@
 [[osp_accounts]]
 ==== Creating OpenStack User Accounts, Projects and Roles
```

Comment:
Add a comma before "and"
Creating OpenStack User Accounts, Projects, and Roles

Reply:
done